# Key points for root_3D_traj_demo
# Each key point: [x, y, z]
# Position in meters

key_points:
  - [1.5, 1.5, 1.0]
  - [2.0, 0.0, 1.0]
  - [1.5, -1.5, 1.0]
  - [0.0, 0.0, 1.0]
  - [-1.5, 1.5, 1.0]
  - [-2.0, 0.0, 1.0]
  - [-1.5, -1.5, 1.0]
  - [0.0, 0.0, 1.0]
